o2a/mappers/action_mapper.py

```diff
@@ -24,22 +25,45 @@
 
 # pylint: disable=abstract-method
 # noinspection PyAbstractClass
-class ActionMapper(BaseMapper):
+from o2a.o2a_libs.property_utils import PropertySet
```

Comment:
@potiuk Can you look at this?

Reply:
Fixed in 2bc950a43ecb5faba2f16ccfd4c50372ccd7db61